README.md

```diff
@@ -45,6 +46,21 @@ e.g `sudo echo "192.168.122.235 ostest-api.test.metalkube.org" >> /etc/hosts`.
 Then you can interact with the k8s API on the bootstrap VM e.g
 `sudo oc status --verbose --config /etc/kubernetes/kubeconfig`.
 
+You can also use the ironic by talking to its API with the openstack client (from the
+virt host)
+`export OS_TOKEN=fake-token`
+`export OS_URL=http://ostest-api.test.metalkube.org:6385/`
```

Comment:
We need to either add ostest-api to /etc/hosts in the scripts or mention above that you need to add it based on the libvirt leases output.

Reply:
We already mention it above,
"""You can then add the IP to the /etc/hosts on the node with the hostname,
e.g `sudo echo "192.168.122.235 ostest-api.test.metalkube.org" >> /etc/hosts`."""